src/ensemble_scheduler.cc

```diff
@@ -1269,7 +1269,13 @@ EnsembleScheduler::Create(
     InferenceServer* const server, const inference::ModelConfig& config,
     std::unique_ptr<Scheduler>* scheduler)
 {
-  scheduler->reset(new EnsembleScheduler(stats_aggregator, server, config));
+  cudaStream_t stream = nullptr;
+#ifdef TRITON_ENABLE_GPU
+  RETURN_IF_ERROR(
+      CreateCudaStream(&stream, server->MinSupportedComputeCapability()));
```

Comment:
Do we need to fail the ensemble creation if it fails to create its own CUDA stream? I think it should still function without dedicated CUDA stream? Unless the error is caused by sticky CUDA error which will fail the inference anyway.

Reply:
I agree. I have removed that from the PR.